knowage-core/src/main/java/it/eng/spagobi/api/AbstractDocumentResource.java

```diff
@@ -88,7 +88,7 @@ public Response insertDocument(String body) {
 		}
 	}
 
-	public Response getDocumentDetails(Object documentIdentifier) {
+	protected Response getDocumentDetails(Object documentIdentifier) {
```

Comment:
If possible, I would prefer private but this is an abstract class, we need a check.

Reply:
This one can't be private, it is being used in DocumentResource.java (two files, one of them is v2).